atlasdb-config/src/main/java/com/palantir/atlasdb/config/AtlasDbConfigs.java

```diff
@@ -47,7 +45,6 @@
             .disable(YAMLGenerator.Feature.WRITE_DOC_START_MARKER));
 
     static {
-        OBJECT_MAPPER.setSubtypeResolver(new DiscoverableSubtypeResolver());
         OBJECT_MAPPER.registerModule(new GuavaModule());
         OBJECT_MAPPER.registerModule(new Jdk7Module());
```

Comment:
Is this required? Modern versions of Jackson do not provide a Jdk7Module because it is supported by the core library.

Reply:
(also unrelated to this change) there's no need for a static block, these calls can be chained onto the ObjectMapper creation

Reply:
@cakofony probably not if you have newer Jackson - though I think that some consumers have lower versions of Jackson, unfortunately.

Reply:
If features from this module are not used we can remove it regardless of the jackson version.

I was under the assumption we held back the jackson version in order to support our favorite internal project which is on a much newer build these days.

Reply:
Did some diving and it looks like we've moved internal applications to newer Jackson. Will remove JDK7 module.

Reply:
Never mind, the update from 2.6.7 in some places only happened recently, will defer that change.